src/pages/index.hbs

```diff
@@ -55,24 +55,34 @@ menuItems:
       <section id="main" class="container">
 
         <section class="box special">
+          <div style="position: relative; float: left">
```

Comment:
do you need position:relative?
Also have you tested on other devices/screen sizes?